source/common/config/well_known_names.cc

```diff
@@ -64,30 +64,29 @@ TagNameValues::TagNameValues() {
          ".dynamodb.");
 
   // mongo.[<stat_prefix>.]collection.[<collection>.]callsite.(<callsite>.)query.*
-  addRe2(MONGO_CALLSITE, R"(^mongo\.<NAME>\.collection\.<NAME>\.callsite\.((<NAME>)\.)query\.)",
-         ".collection.");
+  addTokenized(MONGO_CALLSITE, "mongo.*.collection.*.callsite.$.query.**", "callsite");
```

Comment:
I wanted to ask if it's ok to specify a substring which is located farther from the string beginning, but after having looked at how substrings are matched in the tag extractor I see it doesn't matter now. I like that.